module-types/albis-elcon/WANX-4VDSL.yaml

```diff
@@ -0,0 +1,8 @@
+---
+manufacturer: albis-elcon
+model: WANX-4VDSL
```

Comment:
According to the PDF you provided, it appears that part number `900421` is not `model: WANX-4VDSL`, it is actually `model: WANX-4VDSL-Extender`. Please update the model and file name accordingly

Reply:
It's described as being an extender in the datasheet but the model is printed without `-Extender` on the label.

Reply:
Could you upload a picture of the module that shows the model on it?

Reply:
This has nothing to do with believing or not believing you, it is the fact that the datasheet explicitly lists the model as `WANX-4VDSL-Extender`. I also cannot read the language that PDF is in, so if you could tell me a page to look on or send me an English version I would be happy to validate. I simply asked you to upload a picture of the physical device with the model number to show the exact wording printed on it. 

While I completely understand the sentiment you are conveying in your last sentence, we **do not** want inaccurate information for the sake of being fast. Fast is not always efficient or correct. The entire point of NetBox and this Device/Module Library is to act as a source of truth.